advisories/published/2025/JLSEC-0000-mns39g888-52blx.md

```diff
@@ -0,0 +1,24 @@
+```toml
+schema_version = "1.7.3"
+id = "JLSEC-0000-mns39g888-52blx"
+modified = 2025-10-17T17:21:55.880Z
+upstream = ["CVE-2023-29469"]
+references = ["https://gitlab.gnome.org/GNOME/libxml2/-/issues/510", "https://gitlab.gnome.org/GNOME/libxml2/-/releases/v2.10.4", "https://lists.debian.org/debian-lts-announce/2023/04/msg00031.html", "https://security.netapp.com/advisory/ntap-20230601-0006/", "https://gitlab.gnome.org/GNOME/libxml2/-/issues/510", "https://gitlab.gnome.org/GNOME/libxml2/-/releases/v2.10.4", "https://lists.debian.org/debian-lts-announce/2023/04/msg00031.html", "https://security.netapp.com/advisory/ntap-20230601-0006/"]
+
+[[affected]]
+pkg = "XML2_jll"
+ranges = ["< 2.10.4+0", ">= 2.11.5+0, < 2.12.0+0"]
```

Comment:
This is a little unexpected, but I suppose it's ok.  This advisory describes `libxml2 < 2.10.4`.  We have an unknown version here:

```toml
"2.10.3+0" = {libxml2 = "2.10.3"}
"2.10.4+0" = {libxml2 = "2.10.4"}
"2.11.5+0" = {libxml2 = "*"}
"2.12.0+0" = {libxml2 = "2.12.0"}
"2.12.2+0" = {libxml2 = "2.12.2"}
```

I was thinking that our version comparison would assume that `2.11.5+0` contains some libxml `>= 2.10.4, <= 2.12.0`... which would then imply that `v2.11.5+0` should _not_ be vulnerable.  But obviously that didn't happen here.  It's just fine for this to be less conservative, but might be good to double-check the logic.